src/webview/create-component/pages/fromLocalCodebase.tsx

```diff
@@ -1,26 +1,20 @@
 import { Button, Divider, FormControl, InputLabel, MenuItem, Select, TextField, Typography } from '@mui/material';
-import { makeStyles } from '@mui/styles';
 import * as React from 'react';
-import { WorkspaceFolder } from 'vscode';
-import createComponentStyle from "../../common/createComponent.style";
+import { Uri } from 'vscode';
 import { Devfile } from '../../common/devfile';
 import { DevfileListItem } from '../../common/devfileListItem';
 import { DevfileRecommendationInfo } from '../../common/devfileRecommendationInfo';
```

Comment:
Missed this when taking a look at the previous PR, but check the spelling of "languages" in the info box.